packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -139,4 +138,183 @@ startTelemetry(telemetryConfig);
 // Done! Your container telemetry is now being created and sent to your Telemetry Consumer
 ```
 
-You can now run the app and see the telemetry being printed on your console.
\ No newline at end of file
+You can now run the app and see the telemetry being printed on your console.
+
+## <a name="telemetry_visualization">Interpreting telemetry data</a>
+
+This section provides a set of Azure App Insights queries related to collaborative sessions within a Fluid Framework application. It is intended to be used with the telemetry generated from @fluidframework/fluid-telemetry package whose integration steps are outline above.
+
+### Understanding container telemetry
+
+Before we dive into the queries, we will walk through what a “session” or “collaborative session” is in the context of the following queries. Currently, the concept of a "session” or “collaborative session” does not actually exist within telemetry itself. At a high level, we identify a session by finding a set of container telemetry being emitted with the same container ID within a specific time frame.
+
+Going into more detail, A “Session” or “Collaborative Session” is defined as a period in which we see a continuous stream of container telemetry being emitted with a unique container Id. For a given set of telemetry with the same container Id, if this stream of telemetry stops for longer than a specified period of time then we consider that the end of the session. Each session is differentiated from each other because it has no telemetry (no user activity) emitted for a defined amount of time prior to the start and after the end of emitted telemetry. For each of these queries you can also adjust the time gap that defines a session based on your preference. 
+
+> Note: All telemetry being visualized below is generated from clients without any intervention from the server. The accuracy might be impacted due to inherent nature of client telemetry and data being lost due to faulty clients or lack of connectivity before the telemetry is fully pushed out. We recommend not using this telemetry for business metrics, but rather use it for operational metrics and diagnosis of issues.
+
+### Accessing App Insights Portal
+
+Before we can query, we must first navigate to your Azure App Insights telemetry page. To do this, go to your Azure App Insights Instance and Click on the “Logs” tab under Monitoring
+
+![Logs on App Insights Portal](readme_images/telemetry_1.png)
+
+Now, close out the “Queries” pane if it showed up for you and you will be in the view where we can now execute our queries. Note that if you are using the Fluid Azure App Insights logger, your telemetry data will be available in the “customEvents” table. 
```

Comment:
Should this make reference to this package specifically instead of the "Fluid Azure App Insights Logger"? Or is it intentionally referring to the _other_ package, that pushes the internal telemetry?
```suggestion
Now, close out the “Queries” pane if it showed up for you and you will be in the view where we can execute our queries. Note that if you are using the Fluid Azure App Insights logger, your telemetry data will be available in the “customEvents” table. 
```